config/services.yaml

```diff
@@ -14,13 +14,17 @@ services:
 
     App\:
         resource: '../src/*'
-        exclude: '../src/{DependencyInjection,Entity,Form\EventListener,Migrations,Tests,Kernel.php}'
+        exclude: '../src/{Admin,DependencyInjection,Entity,Form\EventListener,Migrations,Tests,Kernel.php}'
```

Comment:
Removed all existing controllers from DI because the "EasyAdminController" class has been removed, of course.

This change will probably be removed when migration will be done.